GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -21,12 +22,14 @@ public class Champion : Unit
         public int Skin { get; set; }
         private long _championHitFlagTimer;
         private uint _playerId;
+        private uint _playerTeamSpecialId;
```

Comment:
Could this be documented a bit? Maybe add a docstring here and on the constructor of `Champion`? It's pretty hard to tell what this does just by the name at least.